apps/system/js/notification_screen.js

```diff
@@ -509,9 +509,18 @@ var NotificationScreen = {
     window.dispatchEvent(event);
 
     // We turn the screen on if needed in order to let
-    // the user see the notification toaster
+    // the user see the notification toaster unless we're on a call
+    var telephony = window.navigator.mozTelephony;
+    var numOpenLines = 0;
+
+    if (telephony) {
+      numOpenLines = telephony.calls.length +
+        (telephony.conferenceGroup.calls.length ? 1 : 0);
+    }
```

Comment:
Feels like this bit of logic is getting repeated all over the place :)

This is totally optional, or can be a follow up, but I think the `DialerAgent` should have an `onCall` method returning `numOpenLines > 0` and we should `Service.registerState('onCall', this)` when we start the `DialerAgent`.

This way we could simplify the `DialerAgent` a tiny bit and simply do a `Service.query('onCall')` here.